### ISSUE TITLE
0.3.0build error

### ISSUE THREAD

**Issue by artiely** [vuejs/vitepress#21]
```
├──.vitepress
│ ├──dist
│ │ ├──_assets
│ │ │ ├──app.b8eb737b.js
│ │ │ ├──common-40001381.js
│ │ │ ├──index.md.9d20daf2.js
│ │ │ ├──index.md.9d20daf2.lean.js
│ │ │ └──style.34674f41.css
│ ├──theme
│ │ ├──index.js
│ │ └──Layout.vue
├──index.md
├──package-lock.json
└──package.json
```
Write according to the document when executing `npx vitepress build`

build error:
 ReferenceError: __DEV__ is not defined
    at createApp (C:\Users\lizhongyao\Desktop\vitepress\node_modules\vitepress\dist\client\app\temp\app.js:433:17)
    at Object.renderPage (C:\Users\lizhongyao\Desktop\vitepress\node_modules\vitepress\dist\node\build\render.js:12:29)
    at Object.build (C:\Users\lizhongyao\Desktop\vitepress\node_modules\vitepress\dist\node\build\build.js:24:28)

**Comment by oles** [comment]
Getting the same error upon following the steps in the readme:

![Screenshot from 2020-06-03 17-26-32](https://user-images.githubusercontent.com/7634050/83656066-63355e80-a5bf-11ea-81fa-69a1a7d0795a.png)

:thinking:

**Comment by oles** [comment]
Tried on node v12.17.0 as well - same thing

**Comment by yyx990803** [comment]
Should be fixed in 0.3.1
